### ISSUE TITLE
Rule Change: complexity and max line/statements rules : add skipConsole option

### ISSUE THREAD

**Issue by saturdaywalkers** [eslint/eslint#17226]
### What rule do you want to change?

complexity, max-statements, max-lines, max-lines-per-function

### What change to do you want to make?

Generate fewer warnings

### How do you think the change should be implemented?

A new option

### Example code

```js
if ( is_debug ) console.info( "dont count towards complexity, line, or statement counts" );

if ( is_debug ) {
  console.info( 'dont count me either' );
  }

console.group( 'or me either' );
```

### What does the rule currently do for this code?

these statements are debug lines just containing a console.* statement.
they would not be counted in complexity or in max-line/max-statement counts for the other rules 

### What will the rule do after it's changed?

complexity and max-line/max-statement counts will be based on code not on amount of console.* statements

### Participation

- [ ] I am willing to submit a pull request to implement this change.

**Comment by Rec0iL99** [comment]
Hi @saturdaywalkers, thanks for the issue. We are currently not adding any new options to stylistic rules due to the policy change stated in the [README](https://github.com/eslint/eslint#stylistic-rule-updates). You can always copy the existing core rule into a custom rule and add new options.
